### ISSUE TITLE
training result with faster-rcnn

### ISSUE THREAD

**Issue by banxiaduhuo** [XiaozhiChen/resnet-generator#4]
@XiaozhiChen Hi, Xiaozhi. I used your program to create the resnet50's train.prototxt and test.protxt, then trained with faster-rcnn. And I got a very low result on voc2007, about 0.47, even lower than ZF model's 0.62. What's the result you got on voc2007? 

Is my solver correct?
base_lr: 0.001
lr_policy: "multistep"
gamma: 0.1
stepvalue: 300000
stepvalue: 500000
display: 20
momentum: 0.9
weight_decay: 0.0001
snapshot: 0

Thank you very much!

**Comment by XiaozhiChen** [comment]
@banxiaduhuo 
A minor bug was fixed in the script.
I got 71.8 mAP on VOC2007 test set using the the [prototxt](https://github.com/XiaozhiChen/resnet-generator/tree/master/prototxt) and [solver](https://github.com/rbgirshick/py-faster-rcnn/blob/master/models/pascal_voc/VGG16/faster_rcnn_end2end/solver.prototxt) with model trained on VOC07 trainval.

Higher AP could be obtained with some other changes.

**Comment by banxiaduhuo** [comment]
@XiaozhiChen Thank you~ I train the ResNet50 and fix all BN+Scale like your fine-tune.prototxt. After 70k iterations, I got a 0.65 mAP on VOC2007. 
The results of previous 0.47 does not fix the BN+Scale, It seems BN must be fixed when fine-tuning. Did you also use the fine-tune.prototxt to train? And how many iterations?

**Comment by XiaozhiChen** [comment]
@banxiaduhuo 
I was using the finetune.prototxt and 70K iterations.

**Comment by banxiaduhuo** [comment]
@XiaozhiChen Thank you very much!
